pages/common/ffmpeg.md

```diff
@@ -17,3 +17,7 @@
 - Convert AVI video to MP4. AAC Audio @ 128kbit, Video @ 1250Kbit:
 
 `ffmpeg -i {{in.avi}} -acodec libfaac -ab 128k -vcodec mpeg4 -b 1250K {{out.mp4}}`
+
+- Extract part of a video between given timestamps. Replace `-to` with `-t` for length instead of end time.  Omit `-to` to copy through end.
```

Comment:
While it's a good example, you're merging two conflicting parameters in your description (`to` and `t`). I'm never a fan of making examples more complicated by squeezing more logic in there. I know it's a tough one and I can see uses for both, but can you try to opt for either option and discard the other?